decidim-admin/config/locales/ca.yml

```diff
@@ -31,9 +34,12 @@ ca:
         dashboard: Tauler de control
         participatory_processes: Processos participatius
         participatory_processes_submenu:
+          attachments: Adjunts
           info: Informació
           process_admins: Administradors del procés
+          settings: Ajustos
```

Comment:
Yeah, if you feel sure about it, just change it in Crowdin. In the future I guess we should just trust this PRs, as the flow happens there.